### ISSUE TITLE
Cannot save entry with address field with "Not null violation"

### ISSUE THREAD

**Issue by askogrand** [barrelstrength/craft-sprout-fields#101]
### Description (screenshots, logs, ...)
![image](https://user-images.githubusercontent.com/503729/75809581-007dbe00-5d57-11ea-8c04-04b41e9f36b8.png)
![image](https://user-images.githubusercontent.com/503729/75809590-0378ae80-5d57-11ea-8402-d1d05b295d81.png)

### Steps to reproduce

1. Installed the craft-sprout-field plugin
2. Added a new address field
3. Save an entry with an address field populated

### Error

```
Integrity constraint violation – yii\db\IntegrityException

SQLSTATE[23502]: Not null violation: 7 ERROR: null value in column "id" violates not-null constraint
DETAIL: Failing row contains (null, 1510, 1, 92, US, OR, Portland, null, 97202, null, 123 Main Street, , 2020-03-03 19:03:50, 2020-03-03 19:03:50, 64eb543a-d59f-4e4b-9a16-312b4a365772).
The SQL being executed was: INSERT INTO "ut_sprout_addresses" ("id", "elementId", "siteId", "fieldId", "countryCode", "administrativeAreaCode", "locality", "dependentLocality", "postalCode", "sortingCode", "address1", "address2", "dateCreated", "dateUpdated", "uid") VALUES (NULL, 1510, 1, 92, 'US', 'OR', 'Portland', NULL, '97202', NULL, '123 Main Street', '', '2020-03-03 19:03:50', '2020-03-03 19:03:50', '64eb543a-d59f-4e4b-9a16-312b4a365772') RETURNING "id"
```

### Additional info

- Craft version: 3.4.9
- PHP version: 7.2.11
- Database driver & version:  PostgreSQL 9.6.17
- Plugins & versions: 
```
Plugins
Architect	2.3.2
Azure Blob Remote Volume	1.0.2
Chunked File Uploads	1.0.1
Cloner	1.1.0
Field Manager	2.1.1
Freeform	3.6.5
Neo	2.6.2
Redactor	2.6.0.1
Sift	dev-master
Sprout Fields	3.6.1
Workflow	1.4.2

Modules
verbb-base	verbb\base\Base
sprout	barrelstrength\sproutbase\SproutBase
sprout-base-fields	barrelstrength\sproutbasefields\SproutBaseFields
```

[web.log](https://github.com/barrelstrength/craft-sprout-fields/files/4283073/web.log)

**Comment by BenParizek** [comment]
Thanks @askogrand. I can recreate using postgres. I'll update this thread when we have a fix.

**Comment by BenParizek** [comment]
@askogrand This is resolved in the v3.6.2 release.
